WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1882,6 +1874,66 @@ extension ReaderStreamViewController: ReaderPostUndoCellDelegate {
 }
 
 
+// MARK: - Select Interests Display
+private extension ReaderStreamViewController {
+    func displaySelectInterestsIfNeeded(_ content: ReaderContent) {
+        guard
```

Comment:
lets keep the style consistent and do the guard in the same line as well as the else